open-craft/opencraft
# [BB-2225] Fix the validation issue in the combined schema

This fixes the issue causing the validation to not work properly in the combined schema though it works as expected for the individual schemas.

Also fixes a minor issue with the `.env.test` file which was causing all the environment variables to not be set correctly.

**Testing instructions**:
* Verify that the new unit tests added for verifying that this issue is fixed, pass.
* Checkout the source branch of this PR.
* Call the `theme_schema_validate()` function with invalid values for the v0 and v1 schemas and verify that the validation fails as expected.
